IntroToCSharp/Pages/Activities/SettingBuilder.razor

```diff
@@ -0,0 +1,150 @@
+@inherits CaptainCoder.Page
+@page "/activities/setting-builder"
+
+<Section Title="Setting Builder">
+    <Text>
+        When we thinking about making a game a lot of things can come to mind. From 
```

Comment:
Avoid using "we" when possible. Maybe something like, "Many things can come to mind when thinking about creating a game."